.circleci/config.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - restore_cache:
           keys:
             - cache-pip
-      - run: pip install --user numpy matplotlib pyqt5 seaborn sphinx_rtd_theme pillow joblib sphinx pytest vtk traits traitsui pyface mayavi memory_profiler ipython pandas
+      - run: pip install --user numpy matplotlib pyqt5 seaborn sphinx_rtd_theme pillow joblib sphinx pytest vtk traits traitsui pyface mayavi memory_profiler ipython pandas git+https://github.com/executablebooks/sphinx-thebelab#egg=sphinx_thebelab
```

Comment:
note: if you're interested in using this for this PR, and if others are +1 on it, then I can make a pip release pretty easily

Reply:
At this stage whilst PR is in draft, I just added this to get in to build. I'd hold off on this PR until _sphinx-thebelab_ is ready, if that's the route that the project wants to go down.